aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,140 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AvatarAccessibilityExample,
+  AvatarDarkModeExample,
+  AvatarDataTableExample,
+  AvatarDataTableIconExample,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+  MarketplaceNavigationalCards,
+} from '../../examples';
+
+## When to use
+
+Avatars are used to enable users to quickly identify an object, whether that be a specific user, account, or other entity.
+
+Avatars are always paired with additional information, such as a user's name, for better clarity. If possible, use an image or logo over initials.
+
+The consistent dimensions of an avatar ensure that readability and scanability of content surrounding the avatar is not compromised. For example, when scanning a [data table column that contains an avatar of a company’s logo](/avatar#datatable) and the company name, the company name stays aligned from one row to the next. 
+
+## When not to use
+Avatars are not the only components that provide visual representation. When applicable, use these over Avatars:
+- Use [Icons](/foundation/icons) when representing objects, actions, programs, or other communicative symbols.
+- Use an [image](/avatar#avatar-vs-image) when the context of the visual does not need to have equal height and width to maintain alignment of its surrounding content.
```

Comment:
This link results is a page not found error

Reply:
fixed